app/views/recipes/new.html.erb

```diff
@@ -0,0 +1,28 @@
+<div class="new-recipe flex flex-col items-center mx-auto mt-8">
+  <h1 class = "text-3xl font-bold mb-4">Recipe App</h1>
+  <h2 class = "text-2xl mb-4">Add a New recipe</h2>
+  <%= form_with model: @recipe, url: recipes_path, method: :post do |form| %>
+    <div class = "mb-4">
+      <%= form.text_field :name, placeholder: 'Recipe Name', class: "rounded" %>
+    </div>
+    <div class = "mb-4">
+      <%= form.text_area :description, placeholder: 'Description', class: "rounded" %>
```

Comment:
[OPTIONAL]
- You added a validation where a user cannot submit the form unless the description is above 20 characters. When a user enters a description of fewer than 0 characters, the form is not submitted and there is no message to display why. I suggest adding an error message for the user.